v1/src/lib.rs

```diff
@@ -906,9 +906,17 @@ fn key_exprs_to_influx_regex(path_exprs: &[&keyexpr]) -> String {
 }
 
 fn clauses_from_parameters(p: &str) -> ZResult<String> {
-    let time_range = TimeRange::from_str(p);
+    let parameters = Parameters::from(p);
```

Comment:
Could you explain (in a comment, for posterity) why calling the function `from_str` does not work? If it used to work, why does it no longer work?

Reply:
From my understanding, it doesn't work anymore because the logic on Zenoh storage manager side changed in `1.0.0`, it now sends the full parameter string (for example `_time=[..now()]`) instead of just `[..now()]`. The storage has to parse the parameter values itself from the parameter string, which is why I use `Parameters::from(&str)`.

Reply:
Testing locally, I can confirm the behavior has changed and `get()` receives the full string of parameters, 
parsing the full parameters here is the correct fix.